jetbrains-ultimate/tst/softwere/aws/toolkits/jetbrains/utils/rules/NodeJsCodeInsightTestFixtureRule.kt

```diff
@@ -48,7 +47,7 @@ class NodeJsCodeInsightTestFixtureRule : CodeInsightTestFixtureRule() {
 }
 
 class NodeJsLightProjectDescriptor : LightProjectDescriptor() {
-    override fun getModuleType(): ModuleType<*> = getWebModule() ?: throw NullPointerException("module not found")
+    override fun getModuleTypeId(): String = getWebModule()?.id ?: throw NullPointerException("module not found")
```

Comment:
Is this compatible with 192?

Reply:
`./gradlew :jetbrains-ultimate:buildPlugin` runs without errors after switching back to 2019.2. There is only Rider incompatible change in `DotNetSamDebugSupport`.

Reply:
Is there a way to make the Rider code work on 192 and 193?

Reply:
It does not looks so, DotNetDebugProcess() API has been changed with no backward compatibility. We will need to update code.

Reply:
I'll check tomorrow with out debugger squad and will back with updated.

Reply:
Compatibility with 192 SDK is restored in latest 193 SDK build.

Reply:
Thanks!